packages/inter-protocol/test/supports.js

```diff
@@ -232,7 +232,7 @@ export const subscriptionKey = subscription => {
 
 /**
  *
- * @param {ERef<{getPublicTopics: () => import('@agoric/notifier').TopicsRecord}>} hasTopics
+ * @param {ERef<{getPublicTopics: () => import('@agoric/zoe/src/contractSupport').TopicsRecord}>} hasTopics
```

Comment:
you're leaving off `/index.js` on purpose, I trust.

Reply:
yes for TS still automatically resolves `/index.js` (Node style)